spec/rails_helper.rb

```diff
@@ -1,5 +1,3 @@
-require 'codeclimate-test-reporter'
```

Comment:
Is this no longer necessary with new versions of codeclimate?

Reply:
I need to double check this. They've changed some stuff and I thought my changes would work but I'm not sure yet. I'll look into it

Reply:
Okay, so yeah `codeclimate-test-reporter` [seems to be deprecated](https://github.com/codeclimate/ruby-test-reporter). 
I've setup the [new reporting methods](https://docs.codeclimate.com/docs/configuring-test-coverage) as well, I think it should be all set now